src/core/Expander/ExpanderTitle/ExpanderTitle.baseStyles.tsx

```diff
@@ -7,67 +7,65 @@ import { absoluteFocus, noMouseFocus } from '../../theme/utils';
 import { ExpanderTitleProps } from './ExpanderTitle';
 
 export const expanderTitleBaseStyles = withSuomifiTheme(
-  ({ theme }: TokensAndTheme & Partial<ExpanderTitleProps>) => {
-    return css`
-      ${element({ theme })}
-      ${font({ theme })('bodyText')}
+  ({ theme }: TokensAndTheme & Partial<ExpanderTitleProps>) => css`
+    ${element({ theme })}
+    ${font({ theme })('bodyText')}
       font-size: ${theme.typography.bodySemiBold};
```

Comment:
Indent seems to be weird here also for the `font-size`

Reply:
Fixed.